src/actions/account.js

```diff
@@ -164,6 +164,29 @@ async function getAccounts(tokens, options) {
   }, Promise.resolve({}));
 }
 
+export const updatedLoggedinAddresses = tokens => async (dispatch, getState) => {
+  const { network: networkConfig, account, settings } = getState();
+  const newTokens = (Array.isArray(tokens)
+    ? tokens
+    : [tokens]
+  ).filter(key => settings.token.list[key]);
+  if (!newTokens.length) return;
+
+  const [error, info] = await to(getAccounts(newTokens, {
+    networkConfig, passphrase: account.passphrase,
+  }));
+  if (error) {
+    dispatch(errorToastDisplayed({ label: getConnectionErrorMessage(error) }));
+  } else {
+    dispatch(accountUpdated({
+      info: {
+        ...account.info,
```

Comment:
Is this line needed?

Reply:
Actually, since the function only fetches the accounts that were changes to true, it's needed so it doesn't erase the previous fetched accounts

Reply:
Ah got it. I was thinking that in https://github.com/LiskHQ/lisk-hub/blob/ae144e2546cec21d90ca9ebb3632b3150c37c890/src/actions/account.js#L148 it's not needed either, but there it's just account of one token, here it can be multiple...

Should this really allow to handle multiple tokens changed? I think even in the future if we have more tokens, the user will still enable one at a time. 

I'm asking because overall this new action feels like many of the actions we were removing in the refactoring of redux.

Reply:
Yeah I updated it to just fetch the updated token, and with that it's not needed to deestructure anymore.